word-tracker/run-download.py

```diff
@@ -22,12 +22,10 @@
 
 API_SERVICE_NAME = 'youtube'
 API_VERSION = 'v3'
-CLIENT_SECRETS_FILE = 'client_secret.json' # Add your youtube api client login file here.
+CLIENT_SECRETS_FILE = 'client secret' # Add your youtube api client login file here.
 
-# This is the channel ID for Mr Beast - replace with your youtube channel id.
-# Don't know your id? Use this tool https://commentpicker.com/youtube-channel-id.php
-YOUTUBE_CHANNEL_ID = "UCX6OQ3DkcsbYNE6H8uQQuVA"
-CAPTIONS_DIR = 'mrbeast'
```

Comment:
Let's not use input.
If we want this to be configurable, let's use command line arguments.
That way you can rerun the command line easily if it fails